src/pages/workspaces/List.js

```diff
@@ -224,7 +227,7 @@ export const WorkspaceList = ajaxCaller(togglesListView('workspaceList')(class W
           }
         })
       ]),
-      h(PageFadeBox, [
+      h(PageFadeBox, { style: { position: 'relative' } }, [
```

Comment:
Adding `position: relative` to the `PageFadeBox` creates a separate stacking context for the fade, so that it displays on top of the rest of the page, rather than underneath other elements.